testing/trino-product-tests/src/main/java/io/trino/tests/hive/TestHiveStorageFormats.java

```diff
@@ -480,6 +480,35 @@ public void testSnappyCompressedParquetTableCreatedInHive()
         onHive().executeQuery("DROP TABLE " + tableName);
     }
 
+    @Test(groups = STORAGE_FORMATS)
+    public void testOrcStructsWithNonLowercaseFields()
```

Comment:
does this fix a regression with hive dereference pushdown? i.e. does the same queries give non-null values with `hive.projection_pushdown_enabled=false` ?

Reply:
Should we assert results with both `hive.projection_pushdown_enabled = {true | false}`?

Reply:
Thanks for the input @phd3 ! The test queries do give the expected non-null values with `hive.projection_pushdown_enabled=false`. Are you thinking we should add a test case to `TestHiveProjectionPushdownIntoTableScan`?

Reply:
Actually, in a sense we're not interested how Hive itself does field dereference pushdown -- this is Trino's engine's job and Trino's engine doesn't need to be (and is not) aligned with Hive's engine.
Testing with `hive.projection_pushdown_enabled=true / false` should be sufficient here.
(unless we assume Hive can have problem reading the data it just wrote, then yes, reading with onHive would be nice, but i don't find it necessary and maybe not worth the CI time)

Reply:
@willmostly can you please run the test queries with both values of projection_pushdown_enabled ?

Reply:
Now testing with and without projection pushdown enabled.